_includes/schedule-date-cards.html

```diff
@@ -0,0 +1,38 @@
+<div class="row column xlarge-9">
```

Comment:
Extracted to a separate template so we don’t have to copy-paste this between the different pages that display those dates.